installer/bundle/bundle_skel.sh

```diff
@@ -558,6 +559,11 @@ do
             shift 1
             ;;
 
+        -a|--id)
```

Comment:
--id is also used for the workspace id -w|--id so we should probably use another string here